### ISSUE TITLE
tripal file uploader blocked by URL redirect

### ISSUE THREAD

**Issue by bradfordcondon** [erasche/docker-tripal#41]
I'm not able to use the tripal file loader on my local machine running this repo, or my cloud in production using a fork of this stack.  

<img width="1113" alt="Screen Shot 2019-08-24 at 5 27 58 PM" src="https://user-images.githubusercontent.com/7063154/63642886-9cd25400-c694-11e9-9dc0-076aa691b3cb.png">

heres the issue i made in core:

https://github.com/tripal/tripal/pull/978

after some messing around with @spficklin  its definitely something to do with the Apache redirect, but i cant figure out how to resolve it...

basically the PUT in the uploader gets redirected from  `PUT /tripal/tripal/upload/0-tripal_importer/amel_cds.fasta/save/0 `
 to `PUT /upload/0-tripal_importer/amel_cds.fasta/save/0`

it seems like the `tripal_apache.conf` file should hold the answer but i just dont get it.  could you provide any insight into how to resolve this?

**Comment by abretaud** [comment]
This redirection stuff is a bit of a mess, there is some conf in https://github.com/galaxy-genome-annotation/docker-tripal/blob/master/tripal_apache.conf but also some magic in https://github.com/galaxy-genome-annotation/docker-tripal/blob/master/etc/tripal/settings.php#L47

Playing with the env vars should help in many cases, but there might be some corner cases that fail. I think most of the complications come from apache serving tripal at `/tripal/` instead of simply serving it at`/`.
It would be good to change it in the image, but it requires a lot of testing, and I won't be available for most of september so...
